csharp/src/Ice/Communicator.cs

```diff
@@ -103,6 +103,8 @@ private class ObserverUpdater : Instrumentation.IObserverUpdater
             set => _defaultContext = value.ToImmutableDictionary();
         }
 
+        /// <summary>This property controls the default endpoint selection policy for proxies with multiple endpoints.
+        /// </summary>
```

Comment:
Does it really control it if you can only access it? 

```suggestion
        /// <summary>Gets the default endpoint selection policy for proxies with multiple endpoints.
        /// </summary>
```